### PR TITLE
IGAPP-1184: Not found error flickering

### PR DESCRIPTION
**Pull request by steffenkleinle**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by sarahsporck — APPROVED** [review]
Tested in firefox and on android.

File: web/src/routes/CategoriesPage.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import moment from 'moment'
-import React, { ReactElement, useCallback, useContext, useEffect, useRef } from 'react'
+import React, { ReactElement, useCallback, useContext } from 'react'
 import { useTranslation } from 'react-i18next'
 import { Link, Navigate, useParams } from 'react-router-dom'
+import usePreviousProp from 'src/hooks/usePreviousProp'
```
Comment (f1sh1918) [inline web/src/routes/CategoriesPage.tsx:5]:
```suggestion
import usePreviousProp from '../hooks/usePreviousProp'
```

File: web/src/routes/EventsPage.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, { ReactElement, useContext } from 'react'
 import { useTranslation } from 'react-i18next'
 import { useNavigate, useParams } from 'react-router-dom'
+import usePreviousProp from 'src/hooks/usePreviousProp'
```
Comment (f1sh1918) [inline web/src/routes/EventsPage.tsx:4]:
```suggestion
import usePreviousProp from '../hooks/usePreviousProp'
```

**Review by f1sh1918 — CHANGES_REQUESTED** [review]
Nice work. couldn't see any flickering issue

Please adjust the wrong imports and re-request review

**Review by f1sh1918 — APPROVED** [review]

**Pull request merged**